src/Device/style.css

```diff
@@ -0,0 +1,317 @@
+
+.ola_device *,
+.ola_device ::after,
+.ola_device ::before,
+.ola_device::after,
+.ola_device::before {
+  box-sizing: border-box;
+  display: block;
+}
+.ola_device {
+  --scale: 0.5;
+  position: relative;
+  z-index: 1;
+}
+.ola_device .ola_device-frame {
+    z-index: 1; 
+  }
+  .ola_device .ola_device-content {
+    background-color: #fff;
+    background-position: center center;
+    background-size: cover;
+    object-fit: cover;
+    object-position: center top;
+    position: relative;
+    background-color: var(--gray-xlight);
+  }
+  .ola_device-text {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    width: inherit;
+    padding:4px;
+    user-select: none;
+    text-align: center;
```

Comment:
We should include some text styling here. For example:
color: var(--gray);
font: var(--callout);

Reply:
👌🏻